src/utilities/network/lookupApi.js

```diff
@@ -6,7 +6,10 @@
 import { SETTINGS_KEYS } from '../../settings';
 import { UIDatabase } from '../../database';
 
-const { SYNC_URL } = SETTINGS_KEYS;
+import { NAME_TYPES } from '../../sync/syncTranslators';
+import { getOrCreateAddress, parseBoolean, parseDate } from '../../sync/incomingSyncUtils';
```

Comment:
should definitely extract these out!

Reply:
Agreed! I think there is a case for extracting out `sync` helpers as well as `type` helpers (we have a lot of run-time type enforcement stuff throughout the app). It's on the list (see below).